### PR TITLE
Add proceedings information

### PR DESCRIPTION
**Pull request by jhale**

### CODE REVIEW

File: conference/pages_2024/proceedings.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+* Henrik Finsberg, Simula Research Laboratory, Norway
+* Jack S. Hale, University of Luxembourg, Luxembourg
+* Marie E. Rognes, Simula Research Laboratory, Norway
+* Matthew W. Scroggs, University College London, UK
+
+## Review
+
+All submissions will be subject to an initial editorial review, and if
+considered of sufficient quality and within the scope of the volume, sent for
+peer-reviewed by two reviewers (single-blind) who can recommend; acceptance,
```
Comment (meg-simula) [inline conference/pages_2024/proceedings.md:57]: *sent for peer-review*

File: conference/pages_2024/proceedings.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+revision or rejection. The final decision on acceptance will be made by the
+editors.
+
+## Format and requirements
+
+* The submission must be associated with a talk given at the FEniCS Conference
```
Comment (meg-simula) [inline conference/pages_2024/proceedings.md:63]: *talk or poster*

File: conference/pages_2024/proceedings.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+  2024.
+
+* Please make sure to include a concise abstract for your contribution.
+
+* Contributions must use and follow the [SpringerBrief LaTeX
+  template](https://www.springer.com/gp/authors-editors/book-authors-editors/your-publication-journey/manuscript-preparation)
+  (following book chapter) and be 6-10 pages including references. Initial
+  submissions should be made in PDF format. Authors will be invited to
+  submitted original imafes and files if accepted in a single `.zip` or
```
Comment (meg-simula) [inline conference/pages_2024/proceedings.md:72]: *images*

File: conference/pages_2024/proceedings.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+  `.tar.gz` archive without a directory hierarchy.
+
+* All chapters are expected to be accompanied by code released under an [open
+  source initiative](https://opensource.org) approved license and uploaded to
+  e.g. GitHub or GitLab alongside an upload to archival quality repository with
```
Comment (meg-simula) [inline conference/pages_2024/proceedings.md:77]: what is "archival quality repository"?

File: conference/pages_2024/proceedings.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+3. *Early December 2024*: Peer-reviews shared and revised manuscripts invited.
+
+4. *15th January 2025*: Deadline for revised submission.
+
+5. *Early February 2025*: Final editorial review and submission to Simula
+   SpringerBrief Editorial team.
+
+## Editors
+
+* Jørgen Dokken, Simula Research Laboratory, Norway
```
Comment (jorgensd) [inline conference/pages_2024/proceedings.md:47]:
```suggestion
* Jørgen S. Dokken, Simula Research Laboratory, Norway
```

File: conference/pages_2024/proceedings.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+* Henrik Finsberg, Simula Research Laboratory, Norway
```
Comment (jorgensd) [inline conference/pages_2024/proceedings.md:48]:
```suggestion
* Henrik N. Finsberg, Simula Research Laboratory, Norway
```

File: conference/pages_2024/proceedings.md [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+* Jack S. Hale, University of Luxembourg, Luxembourg
+* Marie E. Rognes, Simula Research Laboratory, Norway
+* Matthew W. Scroggs, University College London, UK
```
Comment (jorgensd) [inline conference/pages_2024/proceedings.md:51]:
```suggestion
* Matthew W. Scroggs, University College London, United Kingdom
```

File: conference/pages_2024/proceedings.md (hunk 2 above)
Comment (jorgensd) [inline conference/pages_2024/proceedings.md:63]:
```suggestion
* The submission must be associated with a talk or poster given at the FEniCS Conference
```

File: conference/pages_2024/proceedings.md (hunk 3 above)
Comment (jorgensd) [inline conference/pages_2024/proceedings.md:72]:
```suggestion
  submitted original images and files if accepted in a single `.zip` or
```

File: conference/pages_2024/proceedings.md (hunk 1 above)
Comment (jhale) [reply conference/pages_2024/proceedings.md]: Fixed.

**Pull request merged**